modules/mbedtls/Kconfig

```diff
@@ -76,19 +76,23 @@ config MBEDTLS_SSL_MAX_CONTENT_LEN
 	  twice this value will be allocated (on mbedTLS own heap, so the
 	  value of MBEDTLS_HEAP_SIZE should accommodate that).
 
+module = MBEDTLS
+module-str = Log level mbedTLS library debug hook
+source "subsys/logging/Kconfig.template.log_config"
```

Comment:
it's a good thing you are now using the common log configuration template :+1:

But please remove the prompt text here: https://github.com/zephyrproject-rtos/zephyr/blob/30f07ed81f9f68d4c792ba0b062fc4d57a4a8589/modules/mbedtls/Kconfig#L98

else you'll end in the [stuck symbol syndrome](https://docs.zephyrproject.org/latest/build/kconfig/tips.html#stuck-symbols-in-menuconfig-and-guiconfig)

`ERROR` != `2`
![image](https://user-images.githubusercontent.com/9802583/184326308-6ee7dc28-7eb6-49ea-b1d5-c4a04c6798de.png)